addons/account/wizard/accrued_orders.py

```diff
@@ -179,8 +179,18 @@ def _ellipsis(string, size):
                 for order_line in lines:
                     if is_purchase:
                         account = self._get_computed_account(order, order_line.product_id, is_purchase)
-                        amount = self.company_id.currency_id.round(order_line.qty_to_invoice * order_line.price_unit / rate)
-                        amount_currency = order_line.currency_id.round(order_line.qty_to_invoice * order_line.price_unit)
+                        if any(tax.price_include for tax in order_line.taxes_id):
+                            # As included taxes are not taken into account in the price_unit, we need to compute the price_subtotal
+                            price_subtotal = order_line.taxes_id.compute_all(
```

Comment:
could we not simply use the `price_subtotal` from the purchase order line?

Reply:
@h4818 I specified in the commit message that we can not use the `price_subtotal` of the line as it includes the whole quantity and not only the quantity to invoice. I think this is why it was not used before this commit. 

But I may have understood something wrong, of course.

Reply:
Ah my bad, I missed that...
`price_subtotal / product_qty * qty_to_invoice` doesn't work?

Reply:
@h4818 It works, and I did not manage to have rounding issues but maybe there are with specific values.
In the end, it simply seemed cleaner and safer to compute the value like it is supposed to. 

If you want to use your solution I can change the commit

Reply:
Agreed! Thanks :)